src/src/Helpers/ESPEasy_Storage.cpp

```diff
@@ -1709,10 +1793,33 @@ String ClearInFile(SettingsType::Enum settingsType, int index) {
   if (!getAndLogSettingsParameters(read, settingsType, index, offset, max_size)) {
     return getSettingsFileIndexRangeError(read, settingsType, index);
   }
+  #if FEATURE_EXTENDED_CUSTOM_SETTINGS
+  if (SettingsType::Enum::CustomTaskSettings_Type == settingsType) {
+    max_size = DAT_TASKS_CUSTOM_SIZE; // Don't also wipe the external size inside the config.dat file...
+    DeleteExtendedCustomTaskSettingsFile(settingsType, index);
+  }
+  #endif // if FEATURE_EXTENDED_CUSTOM_SETTINGS
+
   const String fname = SettingsType::getSettingsFileName(settingsType);
   return ClearInFile(fname.c_str(), offset, max_size);
 }
 
+#if FEATURE_EXTENDED_CUSTOM_SETTINGS
+bool DeleteExtendedCustomTaskSettingsFile(SettingsType::Enum settingsType, int index) {
+  if ((SettingsType::Enum::CustomTaskSettings_Type == settingsType) && (INVALID_TASK_INDEX !=index)) {
```

Comment:
Why not use `validTaskIndex` here? (and other places checking for `INVALID_TASK_INDEX`)